packages/addons/service/touchscreen/package.mk

```diff
@@ -20,7 +20,7 @@ PKG_NAME="touchscreen"
 PKG_VERSION="1.0"
 PKG_REV="101"
 PKG_ARCH="any"
-PKG_ADDON_PROJECTS="Generic RPi RPi2"
+PKG_ADDON_PROJECTS="Generic RPi RPi2 Amlogic"
```

Comment:
This won't work without the change I posted.

Reply:
I though your change will be done separately?

Reply:
Not sure how I gave that impression!

> If the above works (not able to test it right now) you might as well add it to this PR.

> Tested, seems to work as expected - please add if it now does what you need

:smile:

Reply:
Simple misunderstanding :(
Adding...